htdocs/product/class/api_products.class.php

```diff
@@ -632,16 +639,15 @@ public function getPurchasePrices($id, $ref = '', $ref_ext = '', $barcode = '',
         }
 
         if ($includestockdata) {
-               $this->product->load_stock();
+            $this->product->load_stock();
```

Comment:
I think getting stock need another api function

Reply:
I also think includestockdata parameter is useless and can be removed.